lookout/style/format/__main__.py

```diff
@@ -90,6 +91,23 @@ def add_parser(name, help):
                                     "modified by adding artificial style mistakes.")
     add_bblfsh_arg(robust_parser)
     add_model_args(robust_parser)
+
+    # Generate different styles dataset
+    gen_styles_parser = add_parser("gen-styles-dataset",
```

Comment:
We need to find smth more precise here to not confuse with the evaluation dataset.

"gen-smoke-dataset"

In the theory of testing "smoke tests" mean super important integration tests which indicate the basic system functionality.